ecosystem/python/sdk/CHANGELOG.md

```diff
@@ -2,9 +2,14 @@
 
 All notable changes to the Aptos Python SDK will be captured in this file. This changelog is written by hand for now.
 
-**Note:** Until we cut a 1.0.0 release, the Aptos Python SDK does not follow semantic versioning.
+## 0.7.0
```

Comment:
maybe do a major version? seems like major breaking changes

Reply:
Haha, I was having a conversation with @banool about this. The reason why we remain in version major version zero is so we can be obnoxious like this.

https://semver.org/#spec-item-4

Reply:
haha ok, well at some point we would need to officially release a production version, guess we first need to agree that there is an official aptos developed and maintained python sdk.

Reply:
Nothing official for now, we should revisit the doc you produced and evaluate what more work needs to be done. As much as I would like to be silly here, I think the two criteria are 1) is it fully fledged according to your doc and 2) our commitment level to its state.